DEPENDENCIES

```diff
@@ -11,7 +11,7 @@ maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.14.2,
 maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.14.2, Apache-2.0, approved, #4699
 maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.14.2, Apache-2.0, approved, clearlydefined
 maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.14.2, Apache-2.0, restricted, clearlydefined
```

Comment:
Restricted libraries need an Eclipse IP review request to be opened with the [Dash-tool](https://github.com/eclipse/dash-licenses/blob/master/README.md). Please follow the [description ](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-7-04) and update the file with the request ID. Then we can merge. Then after all the dependencies are approved, can the product be part of the release.